package.json

```diff
@@ -14,7 +14,7 @@
     "doc": "typedoc ./src/index.ts ./src/**/index.ts",
     "lint": "eslint ./{src,tests}/**/*.ts && prettier --list-different ./{src,tests}/**/*.ts",
     "lint:fix": "eslint ./{src,tests}/**/*.ts --fix && prettier --write ./{src,tests}/**/*.ts",
-    "prepublishOnly": "npm run clean && npm run build",
+    "prepare": "npm run clean && npm run build",
```

Comment:
Why in prepare instead of prepublish?

Reply:
Please read this topic: https://stackoverflow.com/questions/17509669/how-to-install-an-npm-package-from-github-directly
TLDR: The goal is to be able to install the dependency in a project without actually publishing a new version. When running `npm i` it still needs to build something and it does so by running the `prepare` script
For the `prepublishOnly` however, I don't know what it's supposed to do so I suggested to @fdianoux to leave both `prepare` and `prepublishOnly` (but maybe you have a better idea)

Reply:
I see, but shouldn't we be **wanting** versions and not have a direct unversioned link to GitHub?
This seems unnecessary to me.
If it's for local development capacities npm/yarn link command is more than enough. https://docs.npmjs.com/cli/v8/commands/npm-link

Reply:
what if we want to deploy a project to test it, and not only modifying locally?

Reply:
then you can do rc versions I think it would be fine still, but as it will pollute the versions history, it's fine to keep the prepare only and have both git and versions accessible